src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -239,10 +239,16 @@ bool clang_c_languaget::typecheck(
 {
   contextt new_context(msg);
 
+  // Add symbols in new_context
+  // The symbols here cover AST nodes from internals and source code
   clang_c_convertert converter(new_context, ASTs, msg);
   if(converter.convert())
     return true;
 
+  // Adjust symbol attributes, such as:
+  //  - overwrite macro type with real type
```

Comment:
what's a macro type?

Reply:
A 'macro type' refers to the `symbol.is_macro` as being used in function `clang_c_adjust::adjust_type`: 
```
515     if(symbol.is_macro)
516       type = symbol.type; // overwrite
```
I'm not 100% understanding 'is_macro', hence just briefly explaining the purpose of adjuster in a general manner.
This comment does appear vague and a little misleading. 

Q: What does 'symbol.is_macro' actually do in the code? What's the meaning of it?